example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="ext/codemirror.css" rel="stylesheet" type="text/css">
```

Comment:
I think we only used `ext` before in case there was no internet connectivity during the demo, I'd say use the cdn links.